src/Tizen.TV.UIControls.Forms/TVESEnumerations.cs

```diff
@@ -0,0 +1,294 @@
+﻿/*
+ * Copyright (c) 2018 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Tizen.TV.UIControls.Forms
+{
+    /// <summary>
+    /// Enumerator for stream type of es stream
+    /// </summary>
+    public enum StreamType

```

Comment:
It seems need a sub namespace or rename, 
StreamType is too common to use
```suggestion
    public enum ESPlayerStreamType
```

Reply:
Fixed it. Thank you.